### DISCUSSION TITLE
cn/2024/07/07/used-car/

### DISCUSSION THREAD

**Discussion by giscus** [hongtaoh/hongtaoh.github.io#168] [Announcements]
# cn/2024/07/07/used-car/

https://hongtaoh.com/cn/2024/07/07/used-car/

**Comment by wangyunzi** [comment]
我记得看到一个博主在Facebook上直接找卖家比较便宜，话说，米国的二手车这么便宜啊

**Reply by hongtaoh** [reply]
那应该是 facebook marketplace. 

>米国的二手车这么便宜啊

我不觉得便宜啊。美国的二手车我觉得比国内贵。

**Comment by EleanorLiu12** [comment]
从博主关于驾考的帖子搜过来的，受益良多，非常感谢！
